R/parsed.R

```diff
@@ -1,3 +1,15 @@
+#' Parse and pre-process character vector
+#'
+#' @description The function obtains detailed parse information for `text` via
+#'   [utils::getParseData] and does some minimal pre-processing by calling
+#'   [add_ws_to_parse_data].
+#' @inheritSection add_ws_to_parse_data Details
```

Comment:
I'm not sure, I prefer smaller functions with dedicated responsibilities. Are the two functions opposites, or is one calling the other, or...?